src/components/GithubIcon.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { library } from '@fortawesome/fontawesome-svg-core';
```

Comment:
This can be grouped with the other `import` from `@fortawesome/fontawesome-svg-core`.

Reply:
Never mind . Just see https://github.com/jeongm-in/webmark/issues/85#issuecomment-510087478